toolbox/fdc3-workbench/src/store/PrivateChannelStore.tsx

```diff
@@ -197,16 +184,65 @@ class PrivateChannelStore {
 		}
 	}
 
-	onAddContextListener(channel: PrivateChannel, handler: (contextType?: string) => void) {
-		return channel.onAddContextListener(handler);
+	onAddContextListener(channel: PrivateChannel) {
+		channel.onAddContextListener(()=>{
+			try{
+				console.log(channel);
+				systemLogStore.addLog({
+					name: "pcAddContextListener",
+					type: "success",
+					value: `A context listener for '[all]' has been added on channel [${channel.id}]`,
+				});
+			} catch (e) {
+				systemLogStore.addLog({
+					name: "pcAddContextListener",
+					type: "error",
+					value: `Failed to add a context listener for '[all]' on channel [${channel.id}]`,
+				});
+			}
+		});
 	}
 
-	onUnsubscribe(channel: PrivateChannel, handler: (contextType?: string) => void) {
-		return channel.onUnsubscribe(handler);
+	onUnsubscribe(channel: PrivateChannel) {
+		channel.onUnsubscribe(()=>{
+			try {
+				systemLogStore.addLog({
+					name: "pcOnUnsubscribe",
+					type: "success",
+					value: `Sucessfully unsubscribed from listener '[all]' for channel [${channel.id}]`,
+				});
+			} catch (e) {
+				systemLogStore.addLog({
+					name: "pcOnUnsubscribe",
+					type: "error",
+					value: `Could not unsubscribed listener '[all]' from channel [${channel.id}]`,
+				});
+			}
+		});		
 	}
 
-	onDisconnect(channel: PrivateChannel, handler: () => void) {
-		return channel.onDisconnect(handler);
+	onDisconnect(channel: PrivateChannel) {
+		channel.onDisconnect(() => {
+			try {
+				console.log(this.channelListeners)
```

Comment:
Is this intentionally left in? If so, please include a comment about its use

Reply:
removed console logs